### PR TITLE
Adds RUN_MODE env and handles it

### PR DESCRIPTION
**Pull request by scriptnull**
https://github.com/Shippable/reqKick/issues/27

### CODE REVIEW

File: reqKick.app.js [hunk 1]
```diff
@@ -8,8 +8,18 @@ var worker = require('./worker.js');
 function setupGlobals() {
   global.who = 'reqKick|reqKick.app.js';
   global.logger = require('./common/logger.js')();
-  // TODO: Add an ENV for this.
-  logger.level = 'debug';
+
+  var runMode = process.env.RUN_MODE;
+  var logLevel;
```
Comment (shrivara) [inline reqKick.app.js:13]: Lets make the default log level as warn, just in case we get an unknown runMode.

File: init/x86_64/Ubuntu_16.04/shippable-reqKick.env.template [hunk 2]
```diff
@@ -1,3 +1,4 @@
 STATUS_DIR="{{STATUS_DIR}}"
 SCRIPTS_DIR="{{SCRIPTS_DIR}}"
 REQEXEC_BIN_PATH="{{REQEXEC_BIN_PATH}}"
+RUN_MODE="{{RUN_MODE}}"
```
Comment (shrivara) [inline init/x86_64/Ubuntu_16.04/shippable-reqKick.env.template:4]: You need this in the init script for aarch as well. Will that be in a separate PR.

**Comment by scriptnull** [comment]
@shrivara will make the required changes and send it as separate PR now. Thanks!
